### PR TITLE
Api 11600 add letter gen test accounts

### PR DESCRIPTION
**Pull request by KevinCranmer**
https://vajira.max.gov/browse/API-11600

We have a handful of test users used in the upcoming VA Letter Generator app that we want documented here.

### CODE REVIEW

File: test_accounts/letter_generator_test_accounts.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Sandbox Test Accounts for VA Letter Generator API
+
+These test users have a variety of synthetic VA records such as service history and disability rating attached to them and can be used to test the [VA Letter Generator API](LINK_TO_LETTER_GENERATOR_DOCS).
```
[the remark this answers is not in the document]
Comment (KevinCranmer) [reply test_accounts/letter_generator_test_accounts.md:3]: This description holds true for VA Letter Generator API as well

File: test_accounts.md [hunk 2]
```diff
@@ -33,3 +33,7 @@ To log in to the sandbox environment using Login.gov, make sure you:
 ## Veteran Verification
 
 #### [Veteran Verification API](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts/verification_test_accounts.md)
+
+## VA Letters
+
+#### [VA Letter Generator API](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/API-11600-AddLetterGenTestAccounts/test_accounts/letter_generator_test_accounts.md)
```
Comment (dbrown-va) [inline test_accounts.md:39]: Will want this to be the non-branch reference.

File: test_accounts.md [hunk 3]
```diff
@@ -33,3 +33,7 @@ To log in to the sandbox environment using Login.gov, make sure you:
 ## Veteran Verification
 
 #### [Veteran Verification API](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/master/test_accounts/verification_test_accounts.md)
+
+## VA Letters
```
Comment (dbrown-va) [inline test_accounts.md:37]: Recommend removing this entry and link as this page is the landing page specific to OAuth test logins.

The test_accounts/letter_generator_test_accounts.md can remain for now and be directly linked as needed until UX review can make the distinction for test data vs OAuth test logins and where each should live.

Comment (dbrown-va) [inline test_accounts.md:37]: Recommend removing this entry and link as this page is the landing page specific to OAuth test logins.

The test_accounts/letter_generator_test_accounts.md can remain for now and be directly linked as needed until UX review can make the distinction for test data vs OAuth test logins and where each should live.

**Review by dbrown-va — APPROVED** [review]
Test data only rather than full OAuth test accounts, so not linked on the parent Test Accounts markdown.

**Pull request merged**